### PR TITLE
docs: consistency

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/api/consistency.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+Ory Network offers two distinct consistency levels: `full` and `eventual`.
+
+### Full Consistency
+
+**Definition**: The `full` consistency level ensures that you receive the freshest data available.
+
+**Characteristics**:
+
+- **Data Freshness**: Full consistency guarantees that you receive the latest updates to the data.
+- **Latency**: However, this level may introduce higher latency to your queries, as the system takes the necessary steps to
```
Comment (vinckr) [inline docs/api/consistency.mdx:22]:
```suggestion
- **Latency**: However, this level introduces higher latency to your queries, as the system takes the necessary steps to
```

dont beat around the bush

File: docs/api/consistency.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  synchronize and deliver the most recent information.
+- **Use Cases**: Use `full` consistency when access to the most up-to-date data is critical, even if it means slightly longer
```
Comment (vinckr) [inline docs/api/consistency.mdx:24]:
```suggestion
- **Use Cases**: Use `full` consistency when access to the most up-to-date data is critical, even if it means longer
```

File: docs/api/consistency.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  response times.
+
+### Eventual Consistency
+
+**Definition**: The `eventual` consistency level prioritizes delivering data as swiftly as possible.
```
Comment (vinckr) [inline docs/api/consistency.mdx:29]:
```suggestion
**Definition**: The `eventual` consistency level delivers data as fast as possible.
```

File: docs/api/consistency.mdx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+**Characteristics**:
+
+- **Data Freshness**: While `eventual` consistency offers faster query response times, the data may be slightly stale, meaning it
+  might not reflect the very latest updates.
```
Comment (vinckr) [inline docs/api/consistency.mdx:34]:
```suggestion
- **Data Freshness**: While `eventual` consistency offers faster query response times, the data can be stale, meaning it
  does not reflect the latest updates.
```

same as above

File: docs/api/consistency.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+- **Latency**: This level minimizes latency, enabling rapid access to data.
+- **Use Cases**: Choose `eventual` consistency when immediate data access is crucial, and you can tolerate data that is
+  momentarily less current.
```
Comment (vinckr) [inline docs/api/consistency.mdx:37]:
```suggestion
- **Use Cases**: Choose `eventual` consistency when immediate data access is crucial, and you can tolerate data that is not current.
```

File: docs/api/consistency.mdx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+## Replication Lag
+
+It's important to note that Ory Network operates with a replication lag of approximately 4.8 seconds. This means that even under
+`full` consistency, there may still be a brief delay in accessing the absolute latest data due to the replication process. In some
+cases, replication lag might be higher.
```
Comment (vinckr) [inline docs/api/consistency.mdx:43]:
```suggestion
`full` consistency, there is still a delay in accessing the latest data due to the replication process. In some
cases, replication lag can be higher.
```

File: docs/api/consistency.mdx [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+## Identity List API Support
+
+Currently, the [Identity List API(../reference/api#tag/identity/operation/listIdentities) in Ory Network supports the utilization
+of consistency controls. This means you can specify the desired consistency level when using the Identity List API, allowing you
+to tailor your queries to your specific requirements.
```
Comment (vinckr) [inline docs/api/consistency.mdx:49]:
```suggestion
The [Identity List API(../reference/api#tag/identity/operation/listIdentities) in Ory Network supports consistency controls. This means you can choose the consistency level when using the Identity List API.
```

File: docs/api/consistency.mdx [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+## Conclusion
+
+In conclusion, understanding the consistency levels offered by Ory's APIs is essential for optimizing query performance in a
+global system where data is distributed across continents. By choosing the appropriate consistency level, you can strike the right
+balance between data freshness and query latency, ensuring that your application functions effectively in the Ory Network
+environment.
```
Comment (vinckr) [inline docs/api/consistency.mdx:56]: I would cut this, it adds nothing IMO.

File: docs/api/consistency.mdx [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+- **Data Freshness**: While `eventual` consistency offers faster query response times, the data may be slightly stale, meaning it
+  might not reflect the very latest updates.
+- **Latency**: This level minimizes latency, enabling rapid access to data.
+- **Use Cases**: Choose `eventual` consistency when immediate data access is crucial, and you can tolerate data that is
+  momentarily less current.
+
+## Replication Lag
+
+It's important to note that Ory Network operates with a replication lag of approximately 4.8 seconds. This means that even under
```
Comment (kmherrmann) [inline docs/api/consistency.mdx:41]:
```suggestion
It's important to note that Ory Network operates with a replication lag of approximately 5 seconds. This means that even under
```

File: docs/api/consistency.mdx [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+## Identity List API Support
+
+Currently, the [Identity List API(../reference/api#tag/identity/operation/listIdentities) in Ory Network supports the utilization
+of consistency controls. This means you can specify the desired consistency level when using the Identity List API, allowing you
+to tailor your queries to your specific requirements.
+
+## Conclusion
+
+In conclusion, understanding the consistency levels offered by Ory's APIs is essential for optimizing query performance in a
```
Comment (kmherrmann) [inline docs/api/consistency.mdx:53]:
```suggestion
Understanding the consistency levels offered by Ory's APIs is essential for optimizing query performance in a
```

**Review by vinckr — CHANGES_REQUESTED** [review]
please headings instead of bolding 🙏 

Removed a lot of the AI speak to make it more readable and clear.

I can also do a style update, lemme know

**Review by kmherrmann — DISMISSED** [review]
minor suggestions, thanks!

File: docs/api/consistency.mdx (hunk 9 above)
Comment (aeneasr) [reply docs/api/consistency.mdx:40]: It is 4.8 seconds though ;)

**Pull request merged**